src/Aquifer.Data/Entities/BibleBookContentEntity.cs

```diff
@@ -22,4 +27,38 @@ public class BibleBookContentEntity : IHasUpdatedTimestamp
 
     [SqlDefaultValue("getutcdate()")]
     public DateTime Updated { get; set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// POCO for deserializing the AudioUrls column.
+    /// </summary>
+    public sealed class AudioUrlsData
```

Comment:
Opinion: I like the `sealed` keyword on most every class to indicate that I didn't design the class with inheritance in mind.  Any future dev could remove it and take on that responsibility.

I only used it here for these JSON POCOs because I didn't see `sealed` elsewhere in the codebase.

Reply:
I'm fine with that. I think it's more useful for codebases that are being used by other teams, since I doubt anyone of us are gonna try to inherit anyway, and if we did we'd just remove it. So either way.